modules/offers/server/controllers/offers.server.controller.js

```diff
@@ -608,6 +608,25 @@ exports.list = function (req, res) {
     }
   }
 
+  // Filter out users that do not share any circles with the authenticated user
+  // and chose to not appear in those searches.
+  const showOnlyInMyCirclesQueries = [
+    { showOnlyInMyCircles: false },
+    { showOnlyInMyCircles: { $exists: false } },
```

Comment:
This is fine with `$exists` — that said, alternatively we could run a query to add this field as `false` to _every_ existing offer in the database, and have the query simpler.